### PR TITLE
188 Button Tweak

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Add more padding to the bottom on button for better optical alignment

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Run `grunt styleguide`
3. Ensure that all three button types have slightly more padding below than above their content

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- #188 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: core/css/decanter-grid.css [hunk 1]
```diff
@@ -1,37 +1,26 @@
 html {
-  -webkit-box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (josephgknox) [reply core/css/decanter-grid.css]: Thank you, sir. `grunt sass` doesn't compile the prefixes. Running `grunt styleguide` did it.

**Comment by josephgknox** [comment]
@sherakama prefix is back.

https://i.gifer.com/K2si.gif

File: core/scss/utilities/mixins/button/_button-big.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@
 @mixin button-big {
   @include font-smoothing;
   @include margin(0.5em 0.5em 0.5em null);
-  @include padding(1.5rem 3rem);
+  @include padding(1.5rem 3rem 2rem);
```
Comment (yvonnetangsu) [inline core/scss/utilities/mixins/button/_button-big.scss]:
```suggestion
  @include padding(1.5rem 3rem 1.8rem);
```

File: core/scss/utilities/mixins/button/_button-primary.scss [hunk 3]
```diff
@@ -10,7 +10,7 @@
 @mixin button-primary {
   @include font-smoothing;
   @include margin(0.5em 0.5em 0.5em null);
-  @include padding(1rem 2rem);
+  @include padding(1rem 2rem 1.25rem);
```
Comment (yvonnetangsu) [inline core/scss/utilities/mixins/button/_button-primary.scss]:
```suggestion
  @include padding(1rem 2rem 1.15rem);
```

File: core/scss/utilities/mixins/button/_button-secondary.scss [hunk 4]
```diff
@@ -10,7 +10,7 @@
 @mixin button-secondary {
   @include font-smoothing;
   @include margin(0.5em 0.5em 0.5em null);
-  @include padding(1rem 2rem);
+  @include padding(1rem 2rem 1.25rem);
```
Comment (yvonnetangsu) [inline core/scss/utilities/mixins/button/_button-secondary.scss]:
```suggestion
  @include padding(1rem 2rem 1.15rem);
```

**Comment by josephgknox** [comment]
@yvonnetangsu Thanks for confirming with John! These work for me.

**Pull request merged**